hatch/app.py

```diff
@@ -140,17 +141,18 @@ async def workspace_file(
 def workspace_release(
     request: Request,
     workspace: str,
-    release: schema.Release,
+    # currently this API accept the osrelease json or the SPA version
+    filelist: Union[schema.Release, schema.FileList],
```

Comment:
Have you see we can union with pipes in recent versions of Python?

Reply:
Not on the current version release-hatch runs on in Level (3.9), sadly!

Reply:
Once we move to using the release-hatch on the VM, we can switch to 3.10/.11

Reply:
damn you 3.9!